src/getUserEmoji.js

```diff
@@ -120,24 +93,19 @@ function getEmojis(regex, text) {
   if (emojis === null) {
     return null
   }
-  let cleanedEmojis = []
-  emojis.forEach(emoji => {
-    let strippedEmoji = emoji.replace(/:/gi, '')
-    // remove skin tones, numbers and aws crap
-    if (
+  let cleanedEmojis = emojis.map(emoji => emoji.replace(/:/gi, '')).filter(
+    strippedEmoji =>
+      // remove skin tones, numbers and aws crap
       !excluded.includes(strippedEmoji) &&
       isNaN(strippedEmoji) &&
       !strippedEmoji.includes('skin-tone')
-    ) {
-      cleanedEmojis.push(strippedEmoji)
-    }
-  })
+  )
   return cleanedEmojis
 }
 
 function getEmojiList(messages) {
   let userIdEmojiMap = {}
-  if (messages && messages.length > 0) {
+  if (messages) {
```

Comment:
The falsy checks here are something to keep an eye on - in what case would these methods receive false/undefined/0, and why do the "do nothing" operation if they do? There might be a valid reason but this is often a sign of an issue elsewhere.